pkg/addons/addon_dashboard.go

```diff
@@ -0,0 +1,89 @@
+package addons
+
+import (
+	"fmt"
+
+	"github.com/xetys/hetzner-kube/pkg/clustermanager"
+)
+
+// DashboardAddon installs dashboard
+type DashboardAddon struct {
+	masterNode   *clustermanager.Node
+	communicator clustermanager.NodeCommunicator
+}
+
+// NewDashboardAddon installs dashboard to the cluster
+func NewDashboardAddon(provider clustermanager.ClusterProvider, communicator clustermanager.NodeCommunicator) ClusterAddon {
+	masterNode, _ := provider.GetMasterNode()
+	return DashboardAddon{masterNode: masterNode, communicator: communicator}
+}
+
+func init() {
+	addAddon(NewDashboardAddon)
+}
+
+// Name returns the addons name
+func (addon DashboardAddon) Name() string {
+	return "dashboard"
+}
+
+// Requires returns a slice with the name of required addons
+func (addon DashboardAddon) Requires() []string {
+	return []string{}
+}
+
+// Description returns the addons description
+func (addon DashboardAddon) Description() string {
+	return "Kubernetes Dashboard"
+}
+
+// URL returns the URL of the addons underlying project
+func (addon DashboardAddon) URL() string {
+	return "https://github.com/kubernetes/dashboard"
+}
+
+// Install performs all steps to install the addon
+func (addon DashboardAddon) Install(args ...string) {
+	node := *addon.masterNode
+	_, err := addon.communicator.RunCmd(node, "kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v1.10.1/src/deploy/recommended/kubernetes-dashboard.yaml")
+	FatalOnError(err)
+
+	serviceAccount := `apiVersion: v1
```

Comment:
you create a new file here. Do you clean up the service account in uninstall? If not, please add it so this addon doesn't leave stuff in the cluster plz

Reply:
fixed